.nvmrc

```diff
@@ -0,0 +1 @@
+18.14.0
```

Comment:
we try to propagate the approach with asdf, instead of `nvm/rbenv`, could you switch to `.tool-versions`

Reply:
I haven't heard about this tool :) I installed it on my machine and works really cool (I like it detects and change node version automatically) 💪 
I added .nvmrc to use node version locally but also to tell render to use our node version.

https://render.com/docs/node-version

I checked and `.nvmrc` works cool if I added `legacy_version_file = yes` to `.asdfrc` file. Is it good approach or do you recommend to use another way to set node version on Render and make `.tool-version` instead of `.nvmrc`?

Reply:
my first though about `asdf` was - 'the name is too complicated, there is no chance I will remember it` and then I wrote it on the keyboard xd

Reply:
Well I would say, it's about how you document it, imho, we should fully switch to asdf and use only `.too-versions`
In the README you can write smth like that:

We use [asdf](https://github.com/asdf-vm/) to set up the local machine faster, and install dependencies like nodes.

requirements:
[asdf](https://github.com/asdf-vm/)

example of asdf setup:
```
brew install asdf
# not you case yet :)
# asdf plugin-add adr-tools
# asdf plugin-add direnv
asdf plugin-add nodejs
asdf install
```

then yarn install ...

Reply:
As for render I usually use a Dockerfile similar to this one: 
```Dockerfile

FROM node:18.12.1-alpine AS base

# Install dependencies
RUN apk add --no-cache \
  bash \
  make \
  curl \
  tzdata \
  vim \
  && npm install -g npm@9.4.1

# Create user and set ownership and permissions as required
RUN adduser -D monte \
  && mkdir /app /home/monte/.ssh \
  && chown -R monte:monte /app /home/monte/.ssh \
  && chmod 0700 /home/monte/.ssh

# Use a directory called /app in which to store this application's files.
WORKDIR /app
USER monte

# colorized pretty print "username pwd $ "
ENV PS1='\[\e]0;\u \w\a\]\[\033[01;32m\]\u \[\033[01;34m\]\w \$\[\033[00m\] '

# Development
FROM base AS development
ENV NODE_ENV=development
COPY --chown=monte:monte ./package.json ./package-lock.json /app/
RUN npm ci

# Preparation for production release
FROM base AS build
ENV NODE_ENV=production
COPY --chown=monte:monte ./package.json ./package-lock.json /app/
RUN npm ci --omit=dev
COPY --chown=monte:monte . /app/
RUN npm run build-extensions

# Production image
FROM base AS production
ENV NODE_ENV=production
COPY --chown=monte:monte --from=build /app ./
CMD ["make", "run-my-node-app"]

```

so I don't have such problem, but you can explicitly set `NODE_VERSION` environment variable, or lock engine in package.json